### DISCUSSION TITLE
Fast refresh with native code?

### DISCUSSION THREAD

**Discussion by wuleninja** [microsoft/react-native-windows#13949] [Q&A] [answered]
Hi,
Can Fast Refresh somehow work when edits are done in native module code (C#)?
Or should any edit lead to stopping / rebuilding / re-launching the RNW app?
Many thanks for all these amazing tools 🙇

**Comment by jonthysell** [comment, marked as answer]
This is a Visual Studio feature that I'm not sure we've ever tested with our (admittedly complicated) setup for supporting C# in RNW. It's possible that some small changes (like changing the impl within a method) might work when running a Debug build if both the app and module are C#. But it's possible that making live changes could break the registration process we have for exposing the native code to JS.
